### PR TITLE
Fix sidebar highlighting for studio recipe

### PR DESCRIPTION
**Pull request by delucis**
#### Description (required)

Fixes a highlighting bug in the docs sidebar

| Before | After |
|---|---|
| image | image |

### CODE REVIEW

**Review by TheOtterlord — COMMENTED** [review]
State is still incorrect when visiting http://localhost:4321/en/recipes/
image

File: src/util/isSubPage.ts [hunk 1]
```diff
@@ -26,22 +26,32 @@ const categoryIndex: Partial<Record<ReturnType<typeof getPageCategory>, string>>
...
  * @param parentSlug The language-less slug for the parent to test against e.g. `'guides/content-collections'`
  */
 export function isSubPage(currentPage: string, parentSlug: string): boolean {
-	// Test 1: do the two pages share a base URL segment?
+	// Test 1: is this page a known, top-level page? Don’t match its parent.
+	for (const slug of topLevelExceptions) {
+		if (currentPage.endsWith('/' + slug) && parentSlug !== slug) {
+			return false;
+		}
+	}
```
Comment (TheOtterlord) [inline src/util/isSubPage.ts:43]: This seemed to work for me
```suggestion
		if (parentSlug === slug && !currentPage.endsWith('/' + slug)) {
			return false;
		}
	}
```

Comment (delucis) [reply src/util/isSubPage.ts:48]: Yeah just pushed up something equivalent to this! Thanks for catching this @TheOtterlord.

**Review by TheOtterlord — APPROVED** [review]
LGTM locally!

**Pull request merged**